pepper-apis/pex-antlr/src/main/antlr4/org/broadinstitute/ddp/pex/lang/Equation.g4

```diff
@@ -0,0 +1,88 @@
+grammar Equation;
+
+file_ : expression* EOF;
+
+expression
+   :  expression  POW expression
+   |  expression  (TIMES | DIV)  expression
+   |  expression  (PLUS | MINUS) expression
+   |  LPAREN expression RPAREN
+   |  (PLUS | MINUS)* atom
+   ;
+
+atom
+   : number
+   | variable
+   ;
+
+number
+   : NUMBER
+   ;
+
+variable
+   : VARIABLE
```

Comment:
I see that the PEX grammar allows us to use constructions like ABC[234] to refer to the specific row. But it's not our case as far as I understand. When you specify the expression, you just need to refer to the question by its stable id. In case of multiple answers/rows we will need to compute the equation for each row correspondingly.

So, in your example, grammar should be the same, but the return value will contain two answers for each child question: 2 lengths, 2 widths, 2 computed areas.

Am I missing anything?